std/container/rbtree.d

```diff
@@ -1368,6 +1399,18 @@ assert(equal(rbt[], [5]));
         return removeKey(toRemove[]);
     }
 
+    ///
+    unittest
```

Comment:
Shit I am still confused by this one. DIP82 should come at some point ;-)

http://wiki.dlang.org/DIP82